__tests__/app.test.js

```diff
@@ -23,24 +23,38 @@ describe('app routes', () => {
     return request(app)
       .post('/api/v1/recipes')
       .send({
-        name: 'cookies',
+        name: 'Cookies',
         directions: [
           'preheat oven to 375',
           'mix ingredients',
           'put dough on cookie sheet',
           'bake for 10 minutes'
-        ]
+        ],
+        ingredients: [
+          {
+            name: 'chocolate chips',
+            amount: '1',
+            measurement: 'cup'
+          }
+        ],
       })
       .then(res => {
         expect(res.body).toEqual({
           _id: expect.any(String),
-          name: 'cookies',
+          name: 'Cookies',
           directions: [
             'preheat oven to 375',
             'mix ingredients',
             'put dough on cookie sheet',
             'bake for 10 minutes'
           ],
+          ingredients: [
+            {
+              name: 'chocolate chips',
+              amount: '1',
```

Comment:
Try changing amount: '1' to just a number instead of a string. So it would look like amount: 1

Reply:
And remember we need to expect mongodb to add an id, so you should add in above your name field a _id: expect.any(String)